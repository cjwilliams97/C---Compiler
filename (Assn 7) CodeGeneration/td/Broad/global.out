Var: x of type int [mem: Global loc: 0 size: 1] [line: 1]
Sibling: 1 Var: b of type bool [mem: Global loc: -1 size: 1] [line: 2]
Sibling: 2 Func: main returns type void [mem: Global loc: 0 size: -2] [line: 4]
.   Child: 1 Compound [mem: None loc: 0 size: -2] [line: 5]
.   .   Child: 1 Assign: = of type int [line: 6]
.   .   .   Child: 0 Id: x of type int [mem: Global loc: 0 size: 1] [line: 6]
.   .   .   Child: 1 Const 666 of type int [line: 6]
.   .   Sibling: 1 Assign: = of type bool [line: 7]
.   .   .   Child: 0 Id: b of type bool [mem: Global loc: -1 size: 1] [line: 7]
.   .   .   Child: 1 Const true of type bool [line: 7]
.   .   Sibling: 2 Call: output of type void [line: 9]
.   .   .   Child: 0 Id: x of type int [mem: Global loc: 0 size: 1] [line: 9]
.   .   Sibling: 3 Call: outputb of type void [line: 10]
.   .   .   Child: 0 Id: b of type bool [mem: Global loc: -1 size: 1] [line: 10]
.   .   Sibling: 4 Return [line: 12]
Offset for end of global space: -2
Number of warnings: 0
Number of errors: 0
------------------------------------
