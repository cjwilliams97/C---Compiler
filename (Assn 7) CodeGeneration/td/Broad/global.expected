Loading file: global.tm
666 T
Bye.
